radio_beam/beam.py

```diff
@@ -429,6 +435,16 @@ def as_kernel(self, pixscale):
                                           self.minor.to(u.deg).value/pixscale,
                                           self.pa.to(u.radian).value)
 
+    def as_struct_element(self, pixscale):
```

Comment:
I did a shortcut here: I think it's `sqrt(4*log(2))`
for circular, using mixed notation:
`A_gaussian = 2 * pi * sigma^2 / (sqrt(8*log(2))^2`
`A_tophat = pi * r^2`
`pi r^2 = 2 * pi * sigma^2 / (sqrt(8*log(2))^2`
`r = sqrt(2)/sqrt(8*log(2)) * sigma`